### DISCUSSION TITLE
预训练模型 G_0.pth 和 D_0.pth在哪里获取？

### DISCUSSION THREAD

**Discussion by bilbillm** [svc-develop-team/so-vits-svc#129] [Q&A]
找了半天没找到...=-=

**Comment by lyk7539511** [comment]
https://huggingface.co/models?sort=modified&search=so-vits-svc-4.0

**Reply by vivagwb** [reply]
![image](https://user-images.githubusercontent.com/14922541/232051031-0fcff67b-7091-4c47-99b4-b55477bb6273.png)
which one？
